pdns/dnsbackend.cc

```diff
@@ -273,7 +273,7 @@ bool DNSBackend::getBeforeAndAfterNames(uint32_t id, const DNSName& zonename, co
   // lcqname=labelReverse(lcqname);
   DNSName dnc;
   string relqname, sbefore, safter;
-  relqname=labelReverse(makeRelative(qname.toStringNoDot(), zonename.toStringNoDot())); // FIXME400
+  relqname=labelReverse(makeRelative(toLower(qname.toStringNoDot()), zonename.toStringNoDot()));
```

Comment:
Oh, sure it could use a lot more... Add some lines to the diff and you know what i mean. And please don't forget to cover your eyes ;)
My idea here was a minimal fix since this is all gone in master.